.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     rules: {
         // TODO: move rules to @tinkoff/eslint-config-angular
         '@typescript-eslint/no-useless-constructor': 'off',
+        '@typescript-eslint/no-inferrable-types': ['error', {ignoreParameters: true}],
```

Comment:
not sure about `ignoreParameters`

Reply:
@waterplea what do you think? do we need it?

`ignoreParameters = false`

![image](https://user-images.githubusercontent.com/12021443/147677595-1c1924cd-5d7c-4c81-9657-baee3d989411.png)

![image](https://user-images.githubusercontent.com/12021443/147677626-eba0dc62-5b45-4546-a8e3-03eb8f9c8725.png)

![image](https://user-images.githubusercontent.com/12021443/147677644-dc13ead5-632a-4c21-a665-525888be1806.png)

Reply:
Let's keep type for parameters so it's clear what function expects at a glance.